### DISCUSSION TITLE
No syntax highlighting

### DISCUSSION THREAD

**Discussion by AdmiralRazorbeard** [KDE/ghostwriter#749] [Q&A]
Hi, 

I love ghostwriter but I don't get why there is no syntax highlighting in code block. Is that a deliberate choice ? Is there a way to activate it ?

**Comment by plegrand1** [comment]
Same question for me.
ghostwriter is really a great application, but it would be really nice to have this possibility.
Thank you
Pascal

**Comment by bdonauba** [comment]
I would really appreciate support for  syntax high lightning in ghostwriter output.

**Comment by AurelienJt** [comment]
Any updates on that?

**Comment by BUCH-The-Wise** [comment]
Came here from searching online for this. Would be great if I didn't have to use VScode to properly view .md files containing code blocks (which kind of eliminates the point of having GhostWriter).

**Reply by HarryLang** [reply]
Same - I found this page after trying to get the syntax highlighting to work.  It would be a fantastic feature to have.

**Comment by DavideMarini** [comment]
After several years is sad that it's not been added, would be a great feature
